### DISCUSSION TITLE
openvino EP backend tests

### DISCUSSION THREAD

**Discussion by rbhuchar** [openvinotoolkit/openvino#22026] [Q&A] [answered]
How do I run ONNX backend compliance tests for openvino_EP?

Instructions @ https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/docs/tests.md provides following command 
pytest <OV_REPO_DIR>/src/bindings/python/tests/test_onnx/test_backend.py

but I do not see test_onnx folder under <OV_REPO_DIR>/src/bindings/python/tests

**Comment by ilya-lavrenov** [comment, marked as answer]
Hi @rbhuchar 
The tests we moved to https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/tests/tests_python

CC @gkrivor to update dev documentation.
